rcgen/src/key_pair.rs

```diff
@@ -101,12 +101,25 @@ impl KeyPair {
 					serialized_der: key_pair_serialized,
 				})
 			},
+			#[cfg(all(feature = "aws_lc_rs", not(feature = "ring")))]
+			SignAlgo::Rsa(sign_alg) => {
+				use aws_lc_rs::encoding::AsDer;
+				let key_pair_doc =
+					RsaKeyPair::generate(crate::ring_like::rsa::KeySize::Rsa2048)._err()?;
+				let key_pair_serialized = key_pair_doc.as_der()._err()?.as_ref().to_vec();
+
+				let key_pair = RsaKeyPair::from_pkcs8(&key_pair_serialized).unwrap();
```

Comment:
Is there a reason that this `unwrap` is safe? probably deserves a comment,  or changing to `?` since this is already a fallible context.

Reply:
The unwrap is safe because it parses something that's just been serialized.

Reply:
Ahh ok, and the serializer and the parser are from the same crate, so that does seem reasonable :+1: 

I wouldn't mind a comment saying that explicitly, but also recognize it's fairly obvious in hindsight so I leave it up to you.

Reply:
Removed the whole line. the construction is required for `EcdsaKeyPair` and `Ed25519KeyPair` because ring [doesn't return `Self`](https://docs.rs/ring/latest/ring/signature/struct.Ed25519KeyPair.html#method.generate_pkcs8) in the generator functions but a `Document` which needs to first be turned into a buffer that you then give to ring to parse it. But for RSA key generation, [the API simpler](https://docs.rs/aws-lc-rs/latest/aws_lc_rs/rsa/struct.KeyPair.html#method.generate), here `Self` is returned